languageServer/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{

```

Comment:
Maybe we should create a `tsconfig.base.json` at root and extend from it so that we share as many options between projects as possible.